frame/sudo/src/extensions/mod.rs

```diff
@@ -0,0 +1,18 @@
+// This file is part of Substrate.
```

Comment:
Why do we need this module? It is very unlikely that we will have that many extensions.

Reply:
Why not

Reply:
Please remove.